### DISCUSSION TITLE
Active Student licence but Copilot expired

### DISCUSSION THREAD

**Discussion by alexandrosmagos** [community/community#43652] [Copilot Conversations]
### Select Topic Area

Bug

### Body

My account is still verified as a student and copilot was working fine until now. Getting the following message in the extension:
"Accounts that participated in the year-long Technical Preview of GitHub Copilot are not eligible for the 60-Day Free Trial."

Any idea why?

**Comment by petesmithofficial** [comment]
I am having the same issue. Came here to raise it.

**Comment by otkrickey** [comment]
I have created a discussion for today's case. Please share your situation. #43634

**Reply by alexandrosmagos** [reply]
Whoops, my bad.

**Comment by otkrickey** [comment]
Solved: #43634
